### ISSUE TITLE
The VM Support to set Blkiotune

### ISSUE THREAD

**Issue by matthewei** [kubevirt/kubevirt#13100]
**Is your feature request related to a problem? Please describe**:
Sometimes, we need to set the io performance of the disk.
such as `read_bytes_sec`, `write_bytes_sec`,`read_iops_sec`,`write_iops_sec`,`total_bytes_sec,`total_bytes_sec_max`,`total_bytes_sec_max_length,`total_iops_sec,omitempty`,`total_iops_sec_max`,`total_iops_sec_max_length`
![image](https://github.com/user-attachments/assets/398bbd6a-b744-437f-a766-399166934c11)

**Describe the solution you'd like**:
I want to set iotune through  vm's CRD. Please have a look the flowing phase.

```yaml
matthew@ubuntu:~|⇒  kubectl explain vm.spec.template.spec.domain.devices.disks --recursive
KIND:     VirtualMachine
VERSION:  kubevirt.io/v1

RESOURCE: disks <[]Object>

DESCRIPTION:
     Disks describes disks, cdroms and luns which are connected to the vmi.

FIELDS:
   blockSize    <Object>
      custom    <Object>
         logical        <integer>
         physical       <integer>
      matchVolume       <Object>
         enabled        <boolean>
   bootOrder    <integer>
   cache        <string>
   cdrom        <Object>
      bus       <string>
      readonly  <boolean>
      tray      <string>
   dedicatedIOThread    <boolean>
   disk <Object>
      bus       <string>
      pciAddress        <string>
      readonly  <boolean>
   io   <string>
   iotune       <Object>
      read_bytes_sec    <integer>
      read_iops_sec     <integer>
      total_bytes_sec   <integer>
      total_bytes_sec_max       <integer>
      total_bytes_sec_max_length        <integer>
      total_iops_sec    <integer>
      total_iops_sec_max        <integer>
      total_iops_sec_max_length <integer>
      write_bytes_sec   <integer>
      write_iops_sec    <integer>
   lun  <Object>
      bus       <string>
      readonly  <boolean>
   name <string>
   serial       <string>
   shareable    <boolean>
   tag  <string>
```
![image](https://github.com/user-attachments/assets/2ef3c3c4-002c-4edd-9733-d54c48edc52f)

**Describe alternatives you've considered**:
Please see the description of the previous stage.

**Additional context**:

**Comment by alicefr** [comment]
@matthewei we don't expose those values in the API, if you want tune those fields, you should use a sidecar.

**Comment by matthewei** [comment]
@alicefr Could you tell me the reason why it can't be exposed from the API? if we expose thoes values in the API and we can set this rules online. But I think you can't realize it if you use sidecar.

**Comment by matthewei** [comment]
The openstack has providerd this interface. Why don't we provide it?

**Comment by alicefr** [comment]
@matthewei this has been asked multiple times in the past, if you search, you will find multiple issues about this.
The main reason is that this expose too specific fields related to storage. Additionally, there is an ongoing [KEP in Kubernetes that would allow to set the blockio tune parameters in the container](cgroup:https://github.com/kubernetes/enhancements/pull/3004). Once, this is merged and implemented, you could define a proper QoS class.

**Comment by matthewei** [comment]
okay, but the KEP support to set `Blkiotune` online?

**Comment by alicefr** [comment]
@matthewei yes, sorry I thought I had put the link, here it is: https://github.com/kubernetes/enhancements/pull/3004

**Comment by matthewei** [comment]
Many Thanks @alicefr ，I still think that setting it directly through CRD is the best, we can specify the relevant settings explicitly, and it does not conflict with KEP. As an operator of DC, I only care about the creation parameters of the virtual machine
